src/contexts/ComponentAdapter/adapters/internalComponentAdapter.tsx

```diff
@@ -1,5 +1,5 @@
-import { HamburgerMenu } from '@/components/Common/UI/HamburgerMenu'
-import type { HamburgerMenuProps } from '@/components/Common/UI/HamburgerMenu/HamburgerMenuTypes'
+import { HamburgerMenu } from '@/components/Common/HamburgerMenu'
+import type { HamburgerMenuProps } from '@/components/Common/HamburgerMenu/HamburgerMenuTypes'
 import type { InternalComponentsContextType } from '@/contexts/ComponentAdapter/useComponentContext'
 
 export const internalComponents: InternalComponentsContextType = {
```

Comment:
Just curious, why are we doing this again? I know I approved the initial menu but i'm spacing about why we're using internal context. We can chat about it tomorrow during our sync.

Reply:
Yeah let's chat about it today. TL;DR some component abstractions make sense for use in our codebase that put together multiple components that have adapters. We don't want consumers to have to provide defaults for these abstractions since they are implementation details that are SDK specific and only compose components that they already have registered adapters for.

talking this through, another option would be to just import hamburger menu directly from our codebase instead of getting it through context